services/director-v2/tests/unit/with_swarm/test_modules_dynamic_sidecar_docker_api.py

```diff
@@ -59,7 +59,7 @@ def dynamic_sidecar_settings(
     monkeypatch.setenv("TRAEFIK_SIMCORE_ZONE", "test_traefik_zone")
     monkeypatch.setenv("SWARM_STACK_NAME", "test_swarm_name")
     monkeypatch.setenv("SIMCORE_SERVICES_NETWORK_NAME", "test_network_name")
-    monkeypatch.setenv("R_CLONE_S3_PROVIDER", "MINIO")
+    monkeypatch.setenv("R_CLONE_PROVIDER", "MINIO")
```

Comment:
I have been wondering at this ```with_swarm``` folder for a while. Actually a lot of the tests use the swarm since they use the pytest_simcore package. why not move these tests in with_dbs or the base folder test/units if they do not use the DB?

Reply:
moved them outside, they do not require any db.